verl/workers/rollout/sglang_rollout/sglang_rollout.py

```diff
@@ -1074,7 +1072,7 @@ def _req_level_generate_sequences(self, prompts: DataProto, **kwargs) -> DataPro
         )
 
         # free cache engine
-        if self.config.free_cache_engine and self._engine is not None and self._tp_rank == 0:
+        if self._engine is not None and self._tp_rank == 0:
```

Comment:
hmmm @HollowMan6 why removing this condition?

Reply:
The `free_cache_engine` configuration will be misinterpreted if we don't remove it here. As `flush_cache()` is to flush the radix cache in SGLang, it's not related to release/resume memory occupation (as per the meaning in vLLM), and we should still flush the radix cache even if release/resume memory occupation is not enabled, to make sure the radix cache is cleared and not reused in the next generation.

cc: @zhaochenyang20 for confirmation.